src/applications/vaos/services/appointment/transformers.js

```diff
@@ -67,7 +68,7 @@ export function transformVAOSAppointment(appt, useFeSourceOfTruthTelehealth) {
   const isUpcoming = appt.future;
   const isCCRequest = appointmentType === APPOINTMENT_TYPES.ccRequest;
   const providers = appt.practitioners;
-  const start = moment(appt.localStartTime, 'YYYY-MM-DDTHH:mm:ss');
```

Comment:
We should file a ticket to remove localStartTime from vets-website and vets-api now that we no longer use it in any of the business logic.